src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder

```diff
@@ -11,7 +11,7 @@ FROM ubuntu:24.04
 ARG ARCH="amd64"
 ARG YQ_ARCH="amd64"
 # PROTOC_ARCH="x86_64" | "s390_64"
-ARG PROTOC_ARCH="x86_64"
+ARG PROTOC_ARCH='$([ "$(uname -m)" = "s390x" ] && echo "s390_64" || echo "x86_64")'
```

Comment:
I think that https://github.com/confidential-containers/cloud-api-adaptor/blob/a6c13f20b0030346b13db6dc59eadf4a4175fdb8/src/cloud-api-adaptor/Makefile#L200 should handle this and that Makefile has support for arm64 in it' logic. Maybe we just need to pass the argument in in the binaries target as well for podvm_binaries.fedora?